README.md

```diff
@@ -27,6 +27,16 @@ Running with Vagrant
 
 You can then ssh to the machine on port 2222 or open a web browser to http://localhost:8080.
 
+Running with Docker & Docker compose
+-----------
+
+1. Install docker & docker-compose
+2. `cp extIds.example.txt ./extIds.txt`
```

Comment:
@jamesRaybould I think this should read `extIds.sample.txt`

Reply:
Even better it doesn't appear to be needed anymore so I've nuked it 🙂